css/base/base.css

```diff
@@ -1,11 +1,13 @@
 :root {
+  --hooverColor: #8490ff;
```

Comment:
Panaudoti galima --primary-color, kurti naujo nereikia. ta pati spalva.

Reply:
naujo nekuriau, nes dariau dar pries spalvu idejima, o kadangi spalvos kodas skyresi, palikau savo is anksciau sukurta, bet kadangi spalvos yra vizualiai panasios tai tada hover pervadinu i primary, o kita trinu.

Reply:
jau yra primary-color nieko kurti nereikia